app/helpers/duration.spec.js

```diff
@@ -0,0 +1,20 @@
+const assert = require('assert')
+
+const duration = require('./duration')
+
+describe('duration', function () {
+    it('should return miliseconds duration', function () {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/UFABCNextOps/ufabc-next-server/pullRequest?prid=8443142)
```suggestion
  it('should return miliseconds duration', function () {
```